packages/wallets/src/features/cfd/modals/ClientVerificationModal/ClientVerificationModal.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { DerivLightUploadPoiIcon } from '@deriv/quill-icons';
+import { Localize, useTranslations } from '@deriv-com/translations';
+import { Text, useDevice } from '@deriv-com/ui';
+import { ModalStepWrapper } from '../../../../components';
+import { getMarketTypeDetails, MARKET_TYPE } from '../../constants';
+import { TModifiedMT5Accounts } from '../../types';
+import { DocumentsList } from './components';
+import './ClientVerificationModal.scss';
+
+type TClientVerificationModal = {
+    account: TModifiedMT5Accounts;
+};
+
+const ClientVerificationModal: React.FC<TClientVerificationModal> = ({ account }) => {
+    const { localize } = useTranslations();
+    const { isMobile } = useDevice();
+    const { title } = getMarketTypeDetails(localize, account.product)[account.market_type || MARKET_TYPE.ALL];
+
+    return (
+        <ModalStepWrapper title={account.is_added ? localize('Verify account') : localize('Create account')}>
+            <div className='wallets-client-verification-modal'>
+                <DerivLightUploadPoiIcon height={128} width={128} />
+                <Text align='center' size={isMobile ? 'md' : 'sm'}>
+                    {account.is_added ? (
+                        <Localize i18n_default_text='Your account needs verification.' />
+                    ) : (
+                        <Localize
+                            i18n_default_text={
+                                'Once your account details are complete, your {{accountName}} account will be ready for you.'
+                            }
+                            values={{ accountName: `MT5 ${title}` }}
+                        />
```

Comment:
Small nitpick, since 'MT5' is constant it can just be included in the default text:
```suggestion
                        <Localize
                            i18n_default_text={
                                'Once your account details are complete, your MT5 {{title}} account will be ready for you.'
                            }
                            values={{ title }}
                        />
```